tagging/requester.py

```
@@ -0,0 +1,128 @@
+        if data["error"] != "OK":
+            print("Error, please investigate")
+            return None
+        # Want to tell the user to manually tag the comic.
+        return data
+
+    def download_img(self, url):
+        try:
+            response = requests.get(url, timeout=3000)
+            response.raise_for_status()
+            image = BytesIO(response.content)
+            return image
```

Ruff (0.14.4)

Consider moving this statement to an `else` block

(TRY300)